cgotorch/cgotorch.h

```diff
@@ -9,8 +9,12 @@ extern "C" {
 
 typedef void *Tensor;
 typedef void *Optimizer;
+typedef struct {
+  Tensor t;
+  char *err;
+} Result;
 Tensor RandN(int rows, int cols, int require_grad);
-Tensor MM(Tensor a, Tensor b);
+Result MM(Tensor a, Tensor b);
```

Comment:
I think the C-conventional way to return more than one values is to have an additional parameter, other than defining `Result` type.

```c
Tensor MM(Tensor a, Tensor b, char** err);
```

Reply:
Done.